frontend/src/pages/purchases/POList.tsx

```diff
@@ -152,50 +119,32 @@ export default function PurchaseOrderList() {
       detailPageURL="/purchase-orders/detail/"
       whitespaceSize={tableWhitespaceSize}
       isNoPagination={isNoPagination}
+      setIsNoPagination={setIsNoPagination}
       isLoading={isLoading}
       setIsLoading={setIsLoading}
       totalNumberOfEntries={numberOfPurchaseOrders}
       setTotalNumberOfEntries={setNumberOfPurchaseOrders}
       rows={purchaseOrders}
       APISortFieldMap={APIPOSortFieldMap}
       callGetAPI={callAPI}
-      paginatorLeft={noPaginationSwitch}
+      paginatorLeft={<></>}
```

Comment:
Same here, once it is made optional all instances of paginatorLeft should be checked and the ones where nothing is passed in can be deleted